CONTRIBUTING.md

```diff
@@ -18,18 +18,32 @@ to do this once to work on any of Facebook's open source projects.
 
 Complete your CLA here: <https://code.facebook.com/cla>
 
-## Issues  
+## Issues
 We use GitHub issues to track public bugs. Please ensure your description is
 clear and has sufficient instructions to be able to reproduce the issue.
 
 Facebook has a [bounty program](https://www.facebook.com/whitehat/) for the safe
 disclosure of security bugs. In those cases, please go through the process
 outlined on that page and do not file a public issue.
 
-## Coding Style  
+## Coding Style
 * 2 spaces for indentation rather than tabs
 * 80 character line length
 
+## Update Carthage Dependencies
+
+1. Add a tag to the target repository
+    - e.g. https://github.com/appium/RoutingHTTPServer/releases
```

Comment:
BTW, this can only be done in repositories that we own. In case of third party repositories one should ask the owner/admin to do it for us if necessary

Reply:
ah, yeah, right